packages/config/src/projects/jovay/jovay.ts

```diff
@@ -26,13 +46,150 @@ export const jovay: ScalingProject = underReviewL2({
       repositories: ['https://github.com/jovaynetwork'],
     },
   },
-  dataAvailability: undefined,
-  proofSystem: {
-    type: 'Validity',
+  proofSystem: undefined,
+  dataAvailability: {
+    layer: DA_LAYERS.ETH_BLOBS_OR_CALLDATA,
+    bridge: DA_BRIDGES.ENSHRINED,
+    mode: DA_MODES.TRANSACTION_DATA_COMPRESSED,
+  },
+  stage: {
+    stage: 'NotApplicable',
+  },
+  config: {
+    escrows: [
+      {
+        address: EthereumAddress('0x922248db4a99bb542539ae7165fb9d7a546fb9f1'),
+        sinceTimestamp: UnixTime(1754392609),
+        tokens: ['ETH'],
+        chain: 'ethereum',
+      },
+      {
+        address: EthereumAddress('0x9869A90FDAc287519E48aff4cCE329907a995162'),
+        sinceTimestamp: UnixTime(1754392609),
+        tokens: ['ETH'],
+        chain: 'ethereum',
+      },
+    ],
+    activityConfig: {
+      type: 'block',
+      startBlock: 1,
+    },
+    trackedTxs: [
+      {
+        uses: [
+          { type: 'liveness', subtype: 'batchSubmissions' },
+          { type: 'l2costs', subtype: 'batchSubmissions' },
+        ],
+        query: {
+          formula: 'functionCall',
+          address: EthereumAddress(
+            '0xe0a28B8918a62edB825055221a1dF12c7C81Bac1',
+          ),
+          selector: '0x0b6ac513',
+          functionSignature:
+            'function commitBatch(uint8 _version, uint256 _batchIndex, uint256 _totalL1MessagePopped)',
+          sinceTimestamp: UnixTime(1758499200),
+        },
+      },
+      {
+        uses: [
+          { type: 'liveness', subtype: 'stateUpdates' },
+          { type: 'l2costs', subtype: 'stateUpdates' },
+        ],
+        query: {
+          formula: 'functionCall',
+          address: EthereumAddress(
+            '0xe0a28B8918a62edB825055221a1dF12c7C81Bac1',
+          ),
+          selector: '0x7256b753',
+          functionSignature:
+            'function verifyBatch(uint8 _prove_type, bytes _batchHeader, bytes32 _postStateRoot, bytes32 _l2MsgRoot, bytes _proof)',
+          sinceTimestamp: UnixTime(1758499200),
+        },
+      },
+    ],
+  },
+  riskView: {
+    stateValidation: {
+      ...RISK_VIEW.STATE_NONE,
+      value: 'TEE attestation',
+      description:
+        'State roots are accepted when attested by a permissioned Intel SGX TEE through the TEEVerifierProxy. The rollup owner can swap the verifier contract and no permissionless challenges exist, so invalid state roots cannot be disputed or verified onchain.',
```

Comment:
the rollup owner that can swap the verifier is more related to exit window than state validation right? i feel like it doesnt belong here